README.md

```diff
@@ -49,6 +49,17 @@ resource "lxd_container" "test1" {
 }
 ```
 
+You can also launch a container directly from a remote image, not locally cached, by referencing the remote image name using the format:
```

Comment:
Cool thanks. No worries - better to be certain.
Also, you're right this needs more documentation - the remotes that are accessible are only the LXD server itself and the 3 pre-canned LXD defaults (images, ubuntu, ubuntu-daily), which needs explanation.